DEVELOPMENT.md

```diff
@@ -114,6 +114,35 @@ We use [Cypress](https://www.cypress.io/) for integration tests and
 - To develop unit tests in watch mode, run: `yarn test:unit:watch`
 
 
+### Running multiple Upstream instances on the same machine
+
+It is possible to launch multiple Upstream instances at the same time for
+testing purposes. At the moment this is only possible in development mode.
+
+```
+mkdir /Users/rudolfs/work/19000
+mkdir /Users/rudolfs/work/20000
+
+# Launch the first instance.
+
+RAD_HOME="/Users/rudolfs/work/19000" \
+RADICLE_UPSTREAM_PROXY_ARGS="--http-listen 127.0.0.1:19000 --peer-listen 0.0.0.0:19000" \
+RADICLE_UPSTREAM_UI_ARGS="backend=localhost:19000" \
+yarn start
+
+# And then in a separate shell, launch the second instance.
+
+RAD_HOME="/Users/rudolfs/work/20000" \
+RADICLE_UPSTREAM_PROXY_ARGS="--http-listen 127.0.0.1:20000 --peer-listen 0.0.0.0:20000" \
+RADICLE_UPSTREAM_UI_ARGS="backend=localhost:20000" \
+yarn start
+```
+
+It is possible to let the OS choose the peer port by setting it to:
+`--peer-listen 0.0.0.0:0`. And if you don't need completely isolated state,
```

Comment:
They're still completely isolated, no? :)

Reply:
No, electron state is shared when using `RAD_PROFILE`.